README.rst

```diff
@@ -1,47 +1,30 @@
 Ray Tutorial
 ============
 
-Setup
------
+To run the tutuorial online, click on the binder link below:
 
-1. Make sure you have Python installed (we recommend using the `Anaconda Python
-   distribution`_). Ray works with both Python 2 and Python 3. If you are unsure
-   which to use, then use Python 3.
+.. image:: https://mybinder.org/badge.svg :target: https://mybinder.org/v2/gh/jzf2101/tutorial/master
```

Comment:
I'd prefer to include a note saying that this is experimental and that some of the exercises may not work.